pkg/utils/logger/zaplogger.go

```diff
@@ -111,7 +111,9 @@ func (logConfig *Configuration) newZapLogger() *structuredLogger {
 
 	logFilePath := logConfig.LogLocation
 
-	if logFilePath != "" && strings.ToLower(logFilePath) != "stdout" {
+	if logFilePath != "" {
```

Comment:
This will just fix cases where log file path is empty but we still need to fix for cases with chaining like cilium and cni is `aws cni`, logFilePath should be set to `AWS_VPC_K8S_PLUGIN_LOG_FILE `

Reply:
@couralex6 - Please attach UT cases.